apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -49,15 +56,87 @@ export const SignedVerdictOverview: React.FC = () => {
     selectedSharingInstitutionId,
     setSelectedSharingInstitutionId,
   ] = useState<ValueType<ReactSelectOption>>()
+  const [
+    requestCourtRecordSignatureResponse,
+    setRequestCourtRecordSignatureResponse,
+  ] = useState<RequestSignatureResponse>()
+  const [
+    courtRecordSignatureConfirmationResponse,
+    setCourtRecordSignatureConfirmationResponse,
+  ] = useState<SignatureConfirmationResponse>()
 
   const router = useRouter()
   const { user } = useContext(UserContext)
-  const { updateCase } = useCase()
+  const {
+    updateCase,
+    requestCourtRecordSignature,
+    isRequestingCourtRecordSignature,
+  } = useCase()
 
   const [extendCaseMutation, { loading: isCreatingExtension }] = useMutation(
```

Comment:
I think code (extendCaseMutation, reloadCase) belong in the useCase hook. I don't know about the getCourtRecordSignatureConfirmation. What do you think?

Reply:
Yes, I agree.

Reply:
I moved extendCaseMutation to useCase.
reloadCase uses a lazy query and is a bit tricky. Will leave until later.